.cascade-code/Chapman.edu/_cascade/templates/Modular/1_column.html

```diff
@@ -1,56 +1,60 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" class="no-js" lang="en">
-<head>
-  <meta charset="utf-8"/>
-  <meta content="IE=edge" http-equiv="X-UA-Compatible"/>
-  <title><system-page-title/> | Chapman University</title>
-  <system-page-meta-keywords/>
-  <system-page-meta-description/>
-  <system-page-meta-author/>
-  <system-region name="OG_TAGS"/>
-  <system-region name="META VIEWPORT"/>
-  <!-- Jquery, Typekit, Common Assets -->
-  <system-region name="TYPEKIT"/>
-  <system-region name="JQUERY"/>
-  <system-region name="CASCADE ASSETS"/>
-  <!-- Analytics -->
-  <system-region name="FB_JS_SDK"/>
-  <system-region name="GOOGLE_ANALYTICS"/>
-  <!-- Any Additional Head Code -->
-  <system-region name="ADDITIONAL HEAD"/>
-</head>
-
-<!--[if gt IE 8]><!-->
-<body>
-  <!--<![endif]-->
-  <system-region name="GOOGLE_TAG_MANAGER"/>
-  <system-region name="PAGE WRAPPER OPEN"/>
-  <system-region name="JUMP LINK"/>
-  <div id="fb-root"></div>
-  <system-region name="NO_SCRIPT"/>
-  <div class="bigMasthead smallRotator" id="container">
-    <header>
-      <system-region name="OMNI-NAV"/>
-      <system-region name="NAVIGATION"/>    
-    </header>
-    <div id="main" role="main">
-      <div class="primary-content one-column">
-        <system-region name="MASTHEAD"/>
-        <!-- primary content no longer used but leave here until sure content is reassigned to default -->
-        <system-region name="PRIMARY CONTENT"/>
-        <!-- default region replaces primary content region -->
-        <system-region name="DEFAULT"/>
-      </div>
-    </div>
-    <footer>
-      <system-region name="GLOBAL FOOTER"/>    
-    </footer>
-  </div>
-  <!--[if lt IE 7 ]>
-  <script src="//ajax.googleapis.com/ajax/libs/chrome-frame/1.0.3/CFInstall.min.js"></script>
-  <script>window.attachEvent('onload',function(){CFInstall.check({mode:'overlay'})})</script>
-  <![endif]-->
-  <system-region name="ADDITIONAL BODY AT-END"/>
-  <system-region name="PAGE WRAPPER CLOSE"/>
-</body>
+<html
+	xmlns="http://www.w3.org/1999/xhtml" class="no-js" lang="en">
+	<head>
+		<meta charset="utf-8"/>
+		<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
+		<title>
+			<system-page-title/> | Chapman University
+		</title>
+		<system-page-meta-keywords/>
+		<system-page-meta-description/>
+		<system-page-meta-author/>
+		<system-region name="OG_TAGS"/>
+		<system-region name="META VIEWPORT"/>
+		<!-- Jquery, Typkit, Common Assets -->
```

Comment:
@nicholasnadel not a big deal just a type, supposed to be `Typekit`

Reply:
typos are cool now https://twitter.com/ow/status/1191799104919216130

fixed below